benchmarks/query_benchmarks/query_get/benchmark.py

```diff
@@ -0,0 +1,21 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryGet:
+    def setup(self):
+        bench_setup(migrate=True)
+        for i in range(0, 10):
+            Book(pk=i, title=f"foobar_{i}").save()
+
+    def teardown(self):
+        Book.objects.all().delete()
+
+    def time_query_get(self):
+        # This will succeed
+        Book.objects.get(id=1)
```

Comment:
It would be good to repeat this one too; however it's already over multiple lines and will become quite long to write it out in full. 

Maybe in this case the pragmatic approach is to add a for loop and do this for `range(0,10) like in the setup?